pkg/api/healthcheck.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	lambdaTypes "github.com/aws/aws-sdk-go-v2/service/lambda/types"
+
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+	"github.com/common-fate/apikit/apio"
+	"github.com/common-fate/apikit/logger"
+	"github.com/common-fate/common-fate/pkg/cfaws"
+	"github.com/common-fate/common-fate/pkg/deploy"
+	"github.com/common-fate/common-fate/pkg/storage"
+)
+
+func (a *API) AdminRunHealthcheck(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background()
+
+	log := logger.Get(ctx)
+	dc, err := deploy.LoadConfig(deploy.DefaultFilename)
+	if err != nil {
+		apio.Error(ctx, w, err)
+		return
+	}
+	o, err := dc.LoadOutput(ctx)
```

Comment:
Instead of using load output, in the context of the API you should have a parameter which is initialised at startup, it would be loaded from the environment of teh lambda.

In the infrastructure, you add this environment variable and its value.

We use pkg/config to define the environment variables for the servers/lambda